src/app/js/formats/streamgraph/StreamAdmin.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```

Comment:
As explained in previous comment, please delete this file.

Reply:
@Kmaschta @ThieryMichel  I fully understand and agree with the idea to not put useless code on the repo but I would like to let this file because once the pr is accepted I will directly work on the admin part. So if I delete this file and all the links to it I will have to set it up just after and I think it's counterproductive.

What do you think ?

Reply:
Sure, it's understandable!
But we consider each commit on master might be deployed: actually, it's the case, we deploy to production every few PR merged.
Hence, we won't deploy some dead code.

That said, you can have stashed code on your local env without commiting it.

TLDR: WIP code should not end to prod.

Reply:
Okay I understand, I will do the necessary.